src/Database/Schema/TableSchema.php

```diff
@@ -521,9 +521,21 @@ public function getIndex(string $name): ?array
     }
 
     /**
-     * @inheritDoc
+     * Get the column(s) used for the primary key.
+     *
+     * @return array Column name(s) for the primary key. An
+     *   empty list will be returned when the table has no primary key.
+     * @deprecated 4.0.0 Renamed to getPrimaryKey()
```

Comment:
:+1: This works for me. Having a soft deprecation is safe at this point, we can add a yelling deprecation in 4.1